src/shared/lib/oql/oqlfilter.js

```diff
@@ -174,20 +174,30 @@ export function parsedOQLAlterationToSourceOQL(alteration) {
             } else {
                 ret = ["CNA",alteration.constr_rel,alteration.constr_val].join("");
             }
-            ret += alteration.modifiers.map(function(modifier) { return "_"+modifier; }).join("");
+            if (alteration.modifiers.length > 0) {
+                ret += "_";
+            }
+            ret += alteration.modifiers.join("_");
```

Comment:
maybe doesn't matter but you could put line 180 inside the if statment above, no?

Reply:
yeah, that's true